packages/mobile/src/components/suggested-tracks/components/SuggestedTracks.tsx

```diff
@@ -180,26 +180,28 @@ export const SuggestedTracks = (props: SuggestedTracksProps) => {
 
   return (
     <Tile style={styles.root}>
-      <View style={styles.heading}>
-        <View style={styles.headingText}>
-          <Text fontSize='large' weight='heavy' textTransform='uppercase'>
-            {messages.title}
-          </Text>
+      <TouchableOpacity onPress={handleExpanded}>
+        <View style={styles.heading}>
```

Comment:
Does it make sense to have handleExpanded on both the icon and the row? Maybe just make the row and drop the button icon in favor of just icon? Or do we want the animation on icon for press still? Maybe this is okay for a11y cause we need some button element which this services. Does the handle get called twice when clicking the arrow though?